activesupport/test/subscriber_test.rb

```diff
@@ -24,7 +24,22 @@ def private_party(event)
 
 class TestSubscriber2 < ActiveSupport::Subscriber
   attach_to :doodle
+
+  cattr_reader :events
+
+  def self.clear
+    @@events = []
+  end
+
+  def open_party(event)
```

Comment:
Not sure why this shows up in the diff 🤔

Reply:
Ah, I moved `detach_from:` to be after the method definitions because it was detaching the subscriber prior to actually registering any of its patterns (so technically `detach_from` was doing nothing)